Bitdefender/gravityzone/ingest/parser.yml

```
@@ -103,6 +103,10 @@ stages:
           bitdefender.gravityzone.data.categories: "{{parse_event.message.BitdefenderGZDataCategories}}"
           bitdefender.gravityzone.exploit.type: "{{parse_event.message.BitdefenderGZExploitType}}"
 
+      - set:
+          url.original: "{{parse_event.message.BitdefenderGZApplicationControlType}}://{{parse_event.message.request}}"
+        filter: "{{parse_event.message.get('BitdefenderGZApplicationControlType') != None and parse_event.message.get('request') != None and not parse_event.message.request.startswith('http')}}"
+
```

According to the ECS specification, `url.original` is:

> Unmodified original url as seen in the event source.
>
> Note that in network monitoring, the observed URL may be a full URL, whereas in access logs, the URL  is often just represented as a path.
>
> This field is meant to represent the URL as it was observed, complete or not.

Like [Netskope Events format](https://github.com/SEKOIA-IO/intake-formats/blob/main/Netskope/netskope_events/ingest/parser.yml#L115-L117), sadly, I suggest implementing the parse of `url.domain` manually :/:
```suggestion
      - set:
          url.domain: "{{parse_event.message.request.split('/')[0]}}"
          url.path: "/{{parse_event.message.request.split('?')[0].split('/')[1:] | join('/')}}"
          url.query: "{{parse_event.message.request.split('?')[1]}}"
          url.scheme: "{{parse_event.message.BitdefenderGZApplicationControlType}}"
        filter: "{{parse_event.message.get('request') != None and not parse_event.message.request.startswith('http')}}"


```